core/server/utils/get-local-file-storage-path.js

```diff
@@ -0,0 +1,25 @@
+var utils = require('../utils');
+
+/**
+ * Sanitizes a given URL or path for an image to be readable by the local file storage
+ * Always returns {string} url
+ * @param {string} imagePath
+ * @returns {string} imagePath
+ * @description Takes a url or filepath and returns a filepath with is readable
+ * for the local file storage.
+ */
+function getLocalFileStoragePath(imagePath) {
```

Comment:
How about putting this into `core/server/adapters/storage`?
A utils file which offers a set of functions. e.g. `toStoragePath`.

Reply:
Done @kirrg001 !